rustfft.wai

```diff
@@ -0,0 +1,70 @@
+enum fft-direction {
+    forward,
+    inverse
+}
+
+resource complex {
+    static new-f64: func(re: float64, im: float64) -> complex
+    i: func() -> complex
+}
+
+resource inner-fft {
+    process-with-scratch: func(buffer: list<complex>, scratch: list<complex>)
+    process-outofplace-with-scratch: func(input: list<complex>, output: list<complex>, scratch: list<complex>)
+    get-inplace-scratch-len: func() -> u32
+    get-outofplace-scratch-len: func() -> u32
+    process: func(buffer: list<complex>)
+}
```

Comment:
Note that you only need to add methods here if the user is expected to call them. If `process-with-scratch()` and friends are only ever going to be used internally, we don't need to make the methods public.

```suggestion
resource inner-fft
```